examples/app/logs.rs

```diff
@@ -69,7 +69,7 @@ fn log_once_system() {
         info_once!("logs once per call site, so this works just fine: {}", i);
     }
 
-    // you can also use the 'once!' macro directly, in situations you want do do
+    // you can also use the 'once!' macro directly, in situations you want do
     // something expensive only once within the context of a continuous system.
```

Comment:
missing a "where" i think.

might as well also fix grammar/formatting here.

```suggestion
    // you can also use the `once!` macro directly,
    // in situations where you want to do something expensive only once
    // within the context of a continuous system.
```